src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  } else if (days < 7) {
+    return days * 40 - 20;
+  } else {
+    return days * 40 - 50;
+  }
```

Comment:
what a magic numbers 🪄
lets create variables for discount amount and days and then use them
it is better for managing them from one place and for understanding what do they mean

avoid such nesting `else if` conditions
This code is hard to read and maintain